libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/feedback/FeedbackClickListener.java

```diff
@@ -20,6 +22,11 @@
   @Override
   public boolean onInterceptTouchEvent(RecyclerView rv, MotionEvent motionEvent) {
     View child = rv.findChildViewUnder(motionEvent.getX(), motionEvent.getY());
+    ViewGroup group = (ViewGroup)child;
+    if (group != null && group.getChildCount() > 0 && group.getChildAt(0) instanceof ImageView) {
+      ImageView imageView = (ImageView) group.getChildAt(0);
+      imageView.setPressed(!imageView.isPressed());
```

Comment:
this line can result in NPE

Reply:
Already on it. Changed the way ImageView is being used here